src/tasks/misc.ts

```diff
@@ -1133,6 +1136,32 @@ export const MiscQuest: Quest = {
       freeaction: true,
       limit: { tries: 2 },
     },
+    {
+      name: "Clan Photo Booth Free Kill",
+      after: [],
+      priority: () => Priorities.Free,
+      ready: () =>
+          Clan.getWhitelisted().find(
```

Comment:
Don't make web calls in a `ready` call. Probably just call this once at the start of the script.